src/confluent_kafka/admin/__init__.py

```diff
@@ -380,6 +384,8 @@ def create_acls(self, acls, **kwargs):
         :raises TypeException: Invalid input.
         :raises ValueException: Invalid input.
         """
+        if AdminClient._has_duplicated(acls):
+            raise ValueError("pass only different ACL bindings")
```

Comment:
"different" is a bit ambigious, better say: "duplicate ACL bindings not allowed"